src/shared/components/search-result/search-result.component.html

```diff
@@ -1,53 +1,53 @@
 <table *ngIf="hasResults() || hasDrafts()">
   <caption>
-    <h2 class="heading-h2">{{ caseState ? 'Case List' : 'Search result' }}</h2>
+    <h2 class="heading-h2" id="search-result-heading__text" tabindex="-1">{{ caseState ? 'Case List' : 'Search result' }}</h2>
     <div *ngIf="(hasResults() || hasDrafts())" class="pagination-top"
-         attr.aria-label="{{ getTotalResults() }} results have been found">
-      <span class="text-16">Displaying {{ getFirstResult() }} - {{ getLastResult() }} out of {{ getTotalResults() }} results</span>
+      attr.aria-label="{{ getTotalResults() }} results have been found" role="status">
```

Comment:
@phillip-whitaker - what happens if `getTotalResults()` equals 1? Is it OK to have `1 results have been found`?

Reply:
I think so, I don't really want to complicate it.

Reply:
Ah, that's fair enough - caught my eye, so just thought I'd check.